Update lyric.markdown instructions to add clarity (#29111)
home-assistant/home-assistant.github.io: source/_integrations/lyric.markdown

## Setup

To set up this integration, you first **must** set up a developer account with Honeywell using the same email address as your Resideo account:

1. Go to the [developer site](https://developer.honeywellhome.com) and register with an account.
2. Next, create a [new app](https://developer.honeywellhome.com/user/me/apps/add) via the `My Apps` section.
   1. App Name: You can use any name here, but it must not contain special characters.
   2. Callback URL: `https://my.home-assistant.io/redirect/oauth`

>    3. The app will be approved automatically. Click on the name of the app to view the *Consumer Key* and *Consumer Secret* values which you will need later.

Reviewer: ```suggestion
   3. The app will be approved automatically. To view the **Consumer Key** and **Consumer Secret** values, select the name of the app.  Copy the values, you will need them later.
```

{% include integrations/config_flow.md %}

> The integration configuration will ask for the *Client ID* and *Client Secret* which correspond to the *Consumer* values in the App you created above. See [Application Credentials](/integrations/application_credentials) for more details.

Reviewer: ```suggestion
The integration configuration will ask for the **Client ID** and **Client Secret**, which correspond to the **Consumer** values in the app you created above. See [Application Credentials](/integrations/application_credentials) for more details.
```
